core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -243,7 +245,7 @@ constructor(
         constraintsRepository.updateSystemConstraints(systemConstraints)
 
         currentSettings.value =
-            settingsRepository.defaultCameraAppSettings.first()
+            cameraAppSettings
```

Comment:
We'll also want to apply `tryApplyStabilizationConstraints` and `tryApplyFrameRateConstraints` here when they're implemented (see below comments).